turtlebot_navigation/CMakeLists.txt

```diff
@@ -21,6 +21,43 @@ include_directories(${catkin_INCLUDE_DIRS})
 add_executable(laser_footprint_filter src/laser_footprint_filter.cpp)
 target_link_libraries(laser_footprint_filter ${catkin_LIBRARIES})
 
+# Generate camera param files
+include($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
```

Comment:
If you do a `find_package(catkin REQUIRED COMPONENTS turtlebot_bringup)` you should gain access to the `turtlebot_bringup_DIR` variable.